keyboards/newgame40/keymaps/default/keymap.c

```diff
@@ -0,0 +1,213 @@
+  * |   "  |   ,  |   .  |   P  |   Y  |   F  |   G  |   C  |   R  |   L  |
+  * |------+------+------+------+------+------|------+------+------+------|
+  * |   A  |   O  |   E  |   U  |   I  |   D  |   H  |   T  |   N  |   S  |
+  * |------+------+------+------+------+------+-------------+------+------|
+  * |   ;  |   Q  |   J  |   K  |   X  |   B  |   M  |   W  |   V  |   Z  |
+  * |-------------+------+------+------+------+------+------+------+------'
+  * |Shift |  GUI |Lower | Ctrl |Space | Bksp |Enter |Raise | Left |Right |
+  * `---------------------------------------------------------------------'
+  */
+ [_DVORAK] = LAYOUT( \
+   KC_QUOT,   KC_COMM,  KC_DOT,    KC_P,      KC_Y,       KC_F,     KC_G,      KC_C,     KC_R,      KC_L, \
+   KC_A,      KC_O,     KC_E,      KC_U,      KC_I,       KC_D,     KC_H,      KC_T,     KC_N,      KC_S, \
+   KC_SCLN,   KC_Q,     KC_J,      KC_K,      KC_X,       KC_B,     KC_M,      KC_W,     KC_V,      KC_Z, \
+   KC_LSFT, KC_LGUI, LT(_LOWER,KC_LANG2),  KC_LCTL, KC_SPC,  KC_BSPC, KC_ENT, LT(_RAISE,KC_LANG1),  KC_LEFT, KC_RGHT\
+ ),
+
+ /* Lower
+  * ,---------------------------------------------------------------------.
+  * |  Esc |      |      |      |      |      |      |      |      |      |
+  * |------+------+------+------+------+------|------+------+------+------|
+  * |      |      |      |      |      |      |      |      |      |      |
+  * |------+------+------+------+------+------+-------------+------+------|
+  * |  F13 |  F14 |  F15 |  F16 |  F17 |  F18 | F19  | F20  | F21  | F22  |
+  * |-------------+------+------+------+------+------+------+------+------'
+  * |  F1  | F2   |      | F3   |  F4  |  F5  |  F6  |      |  F7 |   F8  |
+  * `---------------------------------------------------------------------'
+  */
+ [_LOWER] = LAYOUT( \
+   KC_ESC,  _______, _______, _______, _______, _______, _______, _______, _______, _______, \
+   _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, \
+   KC_F13,  KC_F14,  KC_F15,  KC_F16,  KC_F17,  KC_F18,  KC_F19,  KC_F20,  KC_F21,  KC_F22,  \
+   KC_F1,   KC_F2,   _______, KC_F3,   KC_F4,   KC_F5,   KC_F6,   _______, KC_F7,   KC_F8    \
+ ),
+
+ /* Raise
+  * ,---------------------------------------------------------------------.
+  * |   1  |   2  |　 3  |   4  |   5  |   6  |   7  |   8  |   9  |   0  |
+  * |------+------+------+------+------+------|------+------+------+------|
+  * |   !  |   "  |   #  |   $  |   %  |   &  |   '  |  (　 |   )  |   _  |
+  * |------+------+------+------+------+------+-------------+------+------|
+  * |   =  |   ¥  |  {   |   }  |   *  |   ~  |   |  |  <   |   >  |   ?  |
+  * |-------------+------+------+------+------+------+------+------+------'
+  * |  F1  |  Alt |      |   [  |   ]  |   ;  |   :  |      |   \  |   /  |
+  * `---------------------------------------------------------------------'
+  */
+ [_RAISE] = LAYOUT( \
```

Comment:
If you're using `LAYOUTS = ortho_4x10`, then you should use that here: 

```suggestion
 [_RAISE] = LAYOUT_ortho_4x10( \
```